api/namex/services/payment/models/__init__.py

```diff
@@ -1,27 +1,32 @@
 from .abstract import Serializable
 from dataclasses import dataclass, field
+from dataclasses_json import dataclass_json, Undefined
```

Comment:
adding this package adds a 3rd json formatter to the toolset. should we add another?

Reply:
I removed it.